docker-compose.yml

```diff
@@ -38,48 +38,51 @@ services:
     labels:
       io.balena.features.supervisor-api: '1'
 
-  srly-ose-viewer:
-    image: screenly/srly-ose-viewer
+  anthias-viewer:
+    image: screenly/anthias-viewer
     build:
       context: .
       dockerfile: docker/Dockerfile.viewer
     depends_on:
-      - srly-ose-server
+      - anthias-server
     environment:
       - HOME=/data
       - PORT=80
       - NOREFRESH=1
-      - LISTEN=srly-ose-nginx
+      - LISTEN=anthias-nginx
     extra_hosts:
       - "host.docker.internal:host-gateway"
+    networks:
+      default:
+      anthias-extra-1:
```

Comment:
I'm confused about this part. Why do we need this?

Reply:
this is (temporarily) needed so that the viewer service can recognize the NGINX service as `anthias-nginx` or `srly-ose-nginx`